media/css/tabzilla/tabzilla.less

```diff
@@ -621,6 +621,12 @@ html[dir="rtl"] {
        (                min-resolution: 1.5dppx),
        (                min-resolution: 144dpi) {
 
+    // high res tab
+    #tabzilla {
+        background-image: url('/media/img/tabzilla/tab-high-res.png');
+        background-size: 150px 90px;
```

Comment:
Somehow I thought I couldn't use the mixin in `tabzilla.less `. Will use it :+1: